src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```diff
@@ -107,8 +104,55 @@ Copyright (c) .NET Foundation. All rights reserved.
       GeneratePublishBlazorBootJson;
     </GenerateStaticWebAssetsPublishManifestDependsOn>
 
+    <ResolveCompressedFilesDependsOn>
+      $(ResolveCompressedFilesDependsOn);
+      _ResolveBlazorWasmOutputs;
+      _AddBlazorWasmStaticWebAssets;
+    </ResolveCompressedFilesDependsOn>
+
+    <ResolveCompressedFilesForPublishDependsOn>
+      $(ResolveCompressedFilesForPublishDependsOn);
+      ProcessPublishFilesForBlazor;
+      ComputeBlazorExtensions;
+      _AddPublishBlazorBootJsonToStaticWebAssets;
+      _CollectAssetsToCompress;
+    </ResolveCompressedFilesForPublishDependsOn>
+
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <_BlazorBuildGZipCompressDirectory>$(IntermediateOutputPath)build-gz\</_BlazorBuildGZipCompressDirectory>
+    <_BlazorPublishCompressDirectory>$(IntermediateOutputPath)compress\</_BlazorPublishCompressDirectory>
   </PropertyGroup>
 
+  <ItemGroup>
+
+    <!--
+      Compress referenced binaries using GZip during build. This skips files such as the project's assemblies
+      that change from build to build. Runtime assets contribute to the bulk of the download size. Compressing it
+      has the most benefit while avoiding any ongoing costs to the dev inner loop.
+    -->
+    <CompressionConfiguration Include="BlazorBuildGzip">
+      <Stage>Build</Stage>
+      <Format>gzip</Format>
+      <IncludePattern>_framework\**</IncludePattern>
+      <ExcludePattern>_framework\blazor.boot.json</ExcludePattern>
```

Comment:
Is there a reason why we need to exclude `_framework\blazor.boot.json`?

Reply:
It's currently not included in the baselines. Should it be?

Reply:
I am hoping that we can define the patterns like this on the Wasm SDK

```
  <!-- Wire-up compression -->
    <ItemGroup>

      <CompressionConfiguration Include="WebassemblyGzip">
        <Stage>All</Stage>
        <Format>gzip</Format>
        <IncludePattern>_framework\**</IncludePattern>
      </CompressionConfiguration>

      <CompressionConfiguration Include="WebassemblyBrotli">
        <Stage>Publish</Stage>
        <Format>brotli</Format>
        <IncludePattern>_framework\**</IncludePattern>
      </CompressionConfiguration>

    </ItemGroup>
```